docs/examples/KModal/ModalWithSubmit&Cancel.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+
+  <div>
+    <KButton @click="openModal">Click Me</KButton>
+    <p v-if="showMessage">Successfully submitted the Form</p>
```

Comment:
```suggestion
    <p v-if="showMessage">Successfully submitted the form</p>
```

Aligns case to writing guidelines.